packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -107,8 +107,37 @@ class DBot {
         });
     }
 
+    /** Compare stored strategy xml with currently running xml */
+    isStrategyChanged(current_xml_dom, recent_files) {
```

Comment:
Just a thought, how about 
```suggestion
    isStrategyUpdated(current_xml_dom, recent_files) {
```

Reply:
Better!